### DISCUSSION TITLE
The compiler list is no longer showing, so I can't select v0.6.0 (or any other version other than "latest local" :(

### DISCUSSION THREAD

**Discussion by marshallthomson** [smartcontractkit/full-blockchain-solidity-course-py#699] [Q&A] [answered]
Hi all, HELP! (pls)

I'm stuck right at the start. I'm trying to change the compiler version to 0.6.0 but the compiler list is not showing any more, so I can't even see that version, let alone select it. The only option that is showing is the _"latest local version - soljson-v0.8.7+commit.e28d00a7.js"_

I've tried searching for the solution, but nobody seems to know or be able to help. 
https://www.reddit.com/r/ethdev/comments/r4sz4y/list_of_compilers_not_showing/
https://www.reddit.com/r/ethdev/comments/pj4urb/remix_compiler_not_working_in_chrome/

Can anyone here help? I can't proceed with this course unless I can sort this out. :(

**Comment by cromewar** [comment]
Delete all your browser data, cache and and so, and then try again. Also using another browser could be helpful.

**Comment by marshallthomson** [comment]
I already tried Edge — No change. Any other suggestions? Should I contact whoever is behind Remix?

Cheers
________________________________

**Reply by cromewar** [reply]
Really strange, there is a downloadable version of Remix, try using this one. Also is there anything blocking your network config? a firewall o even a  vpn?

**Comment by marshallthomson** [comment]
Nothing blocking it. VPN is off. Others are experiencing it too. Did you see the reddit links?

Cheers
________________________________

**Reply by cromewar** [reply]
Yes I did, the only thing they have in common is they were using a browser based on chromium core (chrome, brave... etc) could you please try using firefox to see if it works for you?

**Reply by marshallthomson** [reply]
Weird. firefox works. I can use that for now. I'd love to stick with Chrome though. I'll reach out to Remix.... thanks very much. :)

**Reply by marshallthomson** [reply]
I actually reloaded it again and it works in Chrome! 

In case it's helpful... I found this here: https://gitter.im/ethereum/remix

> Rob @ryestew 01:02
>Currently, the Solidity Compiler is not loading in Remix because the Solidity server is down.
>The problem is being worked on.
>Until it is fixed, try using: latest local version - soljson v0.8.7.
>We will update when the issue is fixed.
>https://twitter.com/EthereumRemix/status/1478727405196005379

Temporary issue - now working in both Firefox and Chrome.

**Reply by cromewar** [reply]
Well It's weird, but I'm glad is solved now.

**Comment by cromewar** [comment, marked as answer]
Closing this thread as is solved now.
